src/features/instance/log/index.tsx

```diff
@@ -135,164 +133,19 @@ export function Logs() {
 	const resetFilters = async () => {
 		form.reset();
 		setLogFilters({
-			limit: 1000,
-			level: undefined,
-			from: undefined,
-			until: undefined,
-			order: 'desc',
+			limit: '1000',
+			level: 'undefined',
+			from: '',
+			until: '',
+			order: 'asc',
 		});
 	};
 
```

Comment:
When you implement the refresh button, the `useRefreshClick` hook we have will be helpful for mimicking the behavior we have elsewhere!